_posts/2017/09/2017-09-19-rfc.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+authors: ["Kate Hertweck"]
+title: "Request for Comment: Share your thoughts on the future of The Carpentries"
+date: 2017-09-19
+time: "15:00:00"
+category: ["Steering Committee", "Community", "Data Carpentry"] 
+---
+
+**We are now accepting comments on plans related to The Carpentries!**
+
+A blog post [last week](https://software-carpentry.org/blog/2017/09/merger.html) provided 
+history and some context behind the planning still in progress for the eventual merger of 
+Data Carpentry and Software Carpentry into a unified organization, tentatively called The 
+Carpentries. An outline of the planned structure, roles, and responsibilities of The 
+Carpentries is [now available](https://github.com/carpentries/2017Merger), and we request 
+your feedback through a series of RFCs and related GitHub issues by October 6, 2017. 
+
+Requests for Comment ([RFCs](https://www.mediawiki.org/wiki/Requests_for_comment), also 
+called Requests for Public Comment) are a tool used by government groups and other 
+organizations to solicit feedback on planned actions which may affect a broad community. 
+So far we have attempted to keep you apprised of the planning process, but are interested 
+in incorporating community input into the unified vision and plan presented in the following 
+topics:
+
+* RFC1 Organization and responsibilities of The Carpentries
+* RFC2 Board of Directors
+* RFC3 Membership Council (transition from current Software Carpentry Advisory Council)
+* RFC4 Staff
+* RCF5 Financial organisation
+* RFC6 Subcommittees and task forces
+* RFC7 Lesson Organizations
+
+**Please head over to the [GitHub repository](https://github.com/carpentries/2017Merger) 
+and add your comments to relevant issues by October 6, 2017.**
```

Comment:
Could add:

If you have any other comments or questions about the merger, please create an issue in [Carpentries conversations](https://github.com/carpentries/conversations), email .. (maybe Kate? or we could set up an email address governance@carpentries.org or use team@carpentries.org and we could forward things) on if we get them), or fill out this anonymous form. (We could create a simple Google form)